runtime/dart_isolate.cc

```diff
@@ -1117,14 +1262,25 @@ void DartIsolate::OnShutdownCallback() {
   }
 }
 
+void DartIsolate::MessageEpilogue(Dart_Handle result) {
```

Comment:
Rename this to something like OnMessageEpilogue so it will not look like it overrides the `DartState::MessageEpilogue` superclass method